### DISCUSSION TITLE
Why is my printer's default max acceleration different from what's mentioned on the website?

### DISCUSSION THREAD

**Discussion by that-ambuj** [1coderookie/Kobra2NeoInsights#4] [Q&A]
I was going through the website and looking for steps to make it faster. I recently bought the Kobra 2 Neo and didn't change a lot of things in firmware because everything seems to be alright with all the calibration I've done.

I stumbled upon this part [https://1coderookie.github.io/Kobra2NeoInsights/firmware/fw_marlin/#default-max-acceleration](https://1coderookie.github.io/Kobra2NeoInsights/firmware/fw_marlin/#default-max-acceleration) and I noticed the values mentioned on the page are higher than what my printer reports.

Here's what my printer spits back:

```
SENDING:M201
  M201 X2000.00 Y2000.00 Z800.00 E2000.00
>>> M203
SENDING:M203
  M203 X300.00 Y300.00 Z8.00 E100.00
>>> M204
SENDING:M204
  M204 P1000.00 R2000.00 T1000.00
```

and here's what's mentioned on the website

![image](https://github.com/user-attachments/assets/30291a72-fbe1-46fb-b9ab-405d623441e3)

Any idea what the reason could be? 

It's a refurbished printer, could the last owner have done that? But I would assume the refurbishers would've reflashed the stock firmware. Maybe I'll flash it again and let you know.

Edit: The Anycubic Slicers(both versions) use the same settings as my printer reports, which I think was intended. I don't understand why it has higher Z accel and speed than the default config but could explain why the nozzle scratches my prints. Plus it has the default extruder speed of 100 and not 25 as mentioned on the website.

**Comment by that-ambuj** [comment]
Just updated the firmware to the same version, the values have not updated.

I'm speculating that the compiled firmware from the website and marlin source have different configurations.

**Comment by 1coderookie** [comment]
Hmm - I only can say that I took those values from their FW source I linked to, so if it *really* differs to what is set in the end (which I kinda doubt if it's the same FW version - assuming they didn't change any settings with the same version..), then I can't really say sth about it. But to see if that's really the case, could you please query a `M503` and copy&paste the output here?

**Reply by that-ambuj** [reply]
Here's the M503 output

```
>>> M503
SENDING:M503
echo:; Linear Units:
echo:  G21 ; (mm)
echo:; Temperature Units:
echo:  M149 C ; Units in Celsius
echo:; Filament settings (Disabled):
echo:  M200 S0 D1.75
echo:; Steps per unit:
echo:  M92 X80.00 Y80.00 Z400.00 E455.00
echo:; Max feedrates (units/s):
echo:  M203 X300.00 Y300.00 Z5.00 E25.00
echo:; Max Acceleration (units/s2):
echo:  M201 X3000.00 Y3000.00 Z500.00 E10000.00
echo:; Acceleration (units/s2) (P<print-accel> R<retract-accel> T<travel-accel>):
echo:  M204 P3000.00 R3000.00 T3000.00
echo:; Advanced (B<min_segment_time_us> S<min_feedrate> T<min_travel_feedrate> X<max_jerk> Y<max_jerk> Z<max_jerk> E<max_jerk>):
echo:  M205 B20000.00 S0.00 T0.00 X10.00 Y10.00 Z0.30 E5.00
echo:; Home offset:
echo:  M206 X0.00 Y0.00 Z0.00
echo:; Auto Bed Leveling:
echo:  M420 S0 ; Leveling OFF
echo:; Material heatup parameters:
echo:  M145 S0 H180.00 B70.00 F0
echo:  M145 S1 H240.00 B110.00 F0
echo:; Hotend PID:
echo:  M301 P14.42 I0.88 D59.12
echo:; Bed PID:
echo:  M304 P97.10 I1.41 D1675.16
echo:; Power-loss recovery:
echo:  M413 S1 ; ON
echo:; Z-Probe Offset:
echo:  M851 X24.00 Y13.35 Z0.00 ; (mm)
echo:; Input Shaping:
echo:  M593 X F55.77 D0.15
echo:  M593 Y F52.70 D0.15
echo:; Linear Advance:
echo:  M900 K0.06
echo:; Filament load/unload:
echo:  M603 L0.00 U100.00 ; (mm)
echo:; Filament runout sensor:
echo:  M412 S1 ; Sensor ON
echo:; UI Language:
echo:  M414 S0
```

Hmm, now it matches the website specs. I assume the previous values were the result of the slicer gcode overriding these values.

I'm sorry for the hassle but I think it was just the limits (emitted to gcode) set by the slicer not by the firmware.

P.S.: These values were taken after reflashing the firmware again and 'restoring defaults' from the printer. Not sure if it clears any overrides or not.

**Reply by 1coderookie** [reply]
OK, then everything is how it's supposed to be. As mentioned in those specific sections at my site, you have to use M503 to query those settings. You can't query those e.g. M203 settings with the same M203 command. It's also mentioned at the MarlinFW site about those specific commands: https://marlinfw.org/docs/gcode/M203.html
